docs/internals/contributing/writing-code/javascript.txt

```diff
@@ -0,0 +1,19 @@
+==========
+JavaScript
+==========
+
+While most of Django core is Python, the ``admin`` and ``gis`` contrib apps contain JavaScript code.
+
+Please follow these coding standards when writing JavaScript code for inclusion in Django.
+
+Code style
+----------
+
+- Please conform to the indentation style dictated in the ``.editorconfig`` file.  We recommend using a text editor with `EditorConfig`_ support to avoid indentation and whitespace issues.  Most of the JavaScript files use 4 spaces for indentation, but there are some exceptions
+
+- When naming variables in JavaScript, ``camelCase`` is usually preferred over ``underscore_case``.  Different JavaScript files sometimes use a different code style.  Please try to conform to the code style of each file.
```

Comment:
I think we should say "When naming varaibles, use camelCase instead of underscore_case."